package.json

```diff
@@ -35,18 +32,19 @@
     "url": "git+https://github.com/jdiaz5513/capnp-ts.git"
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --hoist",
     "benchmark": "gulp benchmark",
     "build": "gulp build",
-    "ci": "gulp lint && gulp coverage && codecov --disable=gcov",
+    "ci": "gulp lint && gulp coverage && codecov --disable=gcov && cd js-examples && ./test.sh",
```

Comment:
Us windows devs are pretty used to porting those npm scripts to batch to get things working.

Seriously however I don't see much issue with having a small shell script, a small batch script and declaring ourselves fully cross platform.